.github/workflows/test.yml

```diff
@@ -224,7 +236,12 @@ jobs:
           name: "${{ github.sha }}-aa"
         if: "env.GIT_DIFF != ''"
       - name: Run tests with race detector
-        run: cat xaa.txt | xargs go test -mod=readonly -timeout 15m -race -tags='cgo ledger test_ledger_mock'
+        run: cat xaa.txt | xargs go test -mod=readonly -json -timeout 15m -race -tags='cgo ledger test_ledger_mock' > xaa-race-output.txt
+        if: "env.GIT_DIFF != ''"
+      - uses: actions/upload-artifact@v2
```

Comment:
I think a `if: env.GIT_DIFF` is missing here, no? Otherwise it is the only step that will be executed.

Reply:
You may be right - @marbar3778 would this step be executed without it?

Reply:
Ok it seems it does not matter as the jobs don't run. Hence resolving this. Thanks!